extends Control

@onready var _start_button: Button = $VBoxContainer/StartButton
@onready var _tutorial_button: Button = $VBoxContainer/TutorialButton
@onready var _controls_button: Button = $VBoxContainer/ContolsButton
@onready var _quit_button: Button = $VBoxContainer/QuitButton


func _ready():
	_start_button.pressed.connect(_on_start_pressed)
	_tutorial_button.pressed.connect(_on_tutorial_pressed)
	_controls_button.pressed.connect(_on_controls_pressed)
	_quit_button.pressed.connect(_on_quit_pressed)

	_start_button.grab_focus()


func _on_start_pressed() -> void:
	get_tree().change_scene_to_file("res://PlanetSelection.tscn")


func _on_tutorial_pressed() -> void:
	get_tree().change_scene_to_file("res://Tutorial/Tutorial.tscn")


func _on_controls_pressed() -> void:
	get_tree().change_scene_to_file("res://GUI/Controls.tscn")


func _on_quit_pressed() -> void:
	get_tree().quit()
